frontend/src/components/column/const.ts

```diff
@@ -0,0 +1,11 @@
+export const ALLOWED_RESPONSIVE_COLUMN_PROPS = [
+  "colWidth",
```

Comment:
can we change this to columnWidth? just to make it consistent with the other ColumnWidth* vars/types